lib/kubernetes-deploy/runner_task.rb

```diff
@@ -200,5 +204,9 @@ def sync_mediator
     def kubeclient
       @kubeclient ||= build_v1_kubeclient(@context)
     end
+
+    def tags(status)
```

Comment:
nit: can we call this `statsd_tags` like in DeployTask? It's very much a statsd thing.